test/suites/smoke-test-common-solo/test-staking-requests.ts

```diff
@@ -0,0 +1,153 @@
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+import { getBlockArray } from "@moonwall/util";
+import type { ApiPromise } from "@polkadot/api";
+import type { GenericExtrinsic } from "@polkadot/types";
+import type { FrameSystemEventRecord } from "@polkadot/types/lookup";
+import type { AnyTuple } from "@polkadot/types/types";
+import Bottleneck from "bottleneck";
+
+const timePeriod = process.env.TIME_PERIOD ? Number(process.env.TIME_PERIOD) : 1 * 60 * 60 * 1000;
+const timeout = Math.max(Math.floor(timePeriod / 12), 5000);
+const hours = (timePeriod / (1000 * 60 * 60)).toFixed(2);
+
+type BlockFilteredRecord = {
+    blockNum: number;
+    extrinsics: GenericExtrinsic<AnyTuple>[];
+    events: FrameSystemEventRecord[];
+    session;
```

Comment:
let's specify the full list of types, at least for the session - it is number I guess